src/ui/public/chrome/api/__tests__/angular.js

```diff
@@ -0,0 +1,20 @@
+import expect from 'expect.js';
+
+import kbnAngular from '../angular';
+import TabFakeStore from '../../__tests__/_TabFakeStore';
+import { noop } from 'lodash';
+
+describe('Chrome API :: Angular', () => {
+  describe('location helper methods', () => {
+    it('should return the sub app based on the url', () => {
+      const chrome = {
+        getInjected: noop,
+        addBasePath: noop
+      };
+      kbnAngular(chrome, {});
```

Comment:
This is giving me trouble. after i run ```chrome.setupAngular()``` the functions ```getBreadcrumbs``` and ```getFirstPathSegment``` still aren't available.